src/main/java/org/veupathdb/service/eda/compute/plugins/correlationassayassay/CorrelationAssayAssayPlugin.java

```diff
@@ -175,6 +175,16 @@ protected void execute() {
             
       
       // Format inputs for R
+      // !!!!!!! TEMPORARY HACK !!!!!!!
+      // This to let the negative values seen in wgcna eigengenes play in mbio-land until we get a better solution
+      // see https://github.com/VEuPathDB/microbiomeComputations/issues/81
+      connection.voidEval("abundanceDataIdColNames <- names(assay1Data)[grepl('stable_id', names(assay1Data))]");
+      connection.voidEval("abundanceDataIds <- assay1Data[,abundanceDataIdColNames, with=FALSE]");
+      connection.voidEval("assay1Data <- plyr::numcolwise(veupathUtils::rescaleToNonNeg)(assay1Data)");
```

Comment:
Suggestion: since we're adding a value to all entries, it's more like shifting than scaling. Perhaps `shiftToNonNeg` would be more clear?

Reply:
unrelated - is this our first plyr invasion?

Reply:
idk about first.. but one of few certainly